test/support/model_case.ex

```diff
@@ -21,7 +21,18 @@ defmodule NationalVoterFile.ModelCase do
       import Ecto
       import Ecto.Changeset
       import Ecto.Query
+      import NationalVoterFile.Factories
       import NationalVoterFile.ModelCase
+
+      defp assert_error_message(changeset, field, expected_message) do
+        {actual_message, _} = changeset.errors[field]
+        assert actual_message == expected_message
+      end
+
+      defp assert_validation_triggered(changeset, field, type) do
+        {_message, status} = changeset.errors[field]
+        assert status[:validation] == type
+      end
```

Comment:
Good opportunity to change these:

```Elixir
defp error_message?(changeset, field, expected_message) do
  {actual_message, _} = changeset.errors[field]
  actual_message == expected_message
end

defp validation_triggered?(changeset, field, type) do
  {_message, status} = changeset.errors[field]
  status[:validation] == type
end
```

Gives us more flexibility by allowing us to do things like

```Elixir
assert error_message?(changeset, :foo, "bar")
refute error_message?(changeset, :foo, "baz")
```

etc.